src/snowflake/cli/api/exceptions.py

```diff
@@ -229,3 +229,13 @@ def __init__(self, show_obj_query: str):
         super().__init__(
             f"Received multiple rows from result of SQL statement: {show_obj_query}. Usage of 'show_specific_object' may not be properly scoped."
         )
+
+
+class FeatureNotEnabledError(RuntimeError):
```

Comment:
If you want print this message in nice error frame you should inherit from ClickException.